Density/app/src/main/res/layout/facilities_activity.xml

```diff
@@ -99,6 +99,56 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
+                <androidx.cardview.widget.CardView
+                    android:id="@+id/covidPolicyCard"
+                    style="@style/MyCardView"
+                    android:layout_width="fill_parent"
+                    android:layout_height="120dp"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="16dp"
+                    android:layout_marginRight="16dp"
+                    android:layout_marginBottom="8dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/appbar">
+
+                    <ImageView
```

Comment:
I would recommend wrapping all the children of the CardView in a ConstraintLayout, for the following reasons:

- set padding (so the top of image isn't touching the top of the card)
- better alignment tools (i.e., don't have to set margins for each child, don't have to use `layout_gravity`)
- don't have to set absolute width and height for any of the children